apps/desktop/src/lib/utils/url.test.ts

```diff
@@ -25,22 +25,6 @@ describe.concurrent('cleanUrl', () => {
 			'https://github.com/user/repo'
 		);
 	});
-
-	const httpRemoteUrls = ['https://github.com/user/repo.git', 'http://192.168.1.1/user/repo.git'];
-
-	test.each(httpRemoteUrls)('HTTP Remote - %s', (remoteUrl) => {
-		expect(remoteUrlIsHttp(remoteUrl)).toBe(true);
-	});
-
-	const nonHttpRemoteUrls = [
-		'git@github.com:user/repo.git',
-		'ssh://git@github.com:22/user/repo.git',
-		'git://github.com/user/repo.git'
-	];
-
-	test.each(nonHttpRemoteUrls)('Non HTTP Remote - %s', (remoteUrl) => {
-		expect(remoteUrlIsHttp(remoteUrl)).toBe(false);
-	});
```

Comment:
Why don't we want these tests?

Reply:
Because they existed for exactly 1 location in the code asking "is the protocol http?". The if condition can just as well be `protocol.startsWith('http')` rather than having a specialized and tested function.